src/openrct2/audio/Audio.cpp

```diff
@@ -322,11 +322,14 @@ namespace OpenRCT2::Audio
         }
     }
 
-    void StopAll()
+    void StopAll(bool preserveMusic)
```

Comment:
This definitely the wrong approach, now you have everyone pass false except for the one case, also its an odd function name if it says StopAll but at the end doesn't. You probably want a new function something like StopEffects.